acm/odh-edge/apps/tensorflow-housing-app/kustomization.yaml

```diff
@@ -31,6 +31,7 @@ patches:
     kind: Deployment
     metadata:
       name: model-1
+      model-name: tensorflow-housing
```

Comment:
The deployment already gets an `app: <model name>` label through the kustomization file, here: https://github.com/opendatahub-io/ai-edge/blob/main/acm/odh-edge/apps/bike-rental-app/kustomization.yaml#L10

If we need an additional label for `model-name`, then I suggest we add it there, rather than to the Deployment directly like this.